homeassistant/components/light/__init__.py

```diff
@@ -352,7 +352,7 @@ async def async_handle_light_on_service(light, call):
             brightness = light.brightness if light.is_on else 0
 
             if ATTR_BRIGHTNESS_STEP in params:
-                brightness += params.pop(ATTR_BRIGHTNESS_STEP)
+                brightness += params[ATTR_BRIGHTNESS_STEP]
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Already an active discussion on it here: https://github.com/home-assistant/architecture/discussions/603#discussioncomment-1557948

Though not completed.

Reply:
Ok, let's close here then since the discussion is not resolved. There's no point in opening a PR until that's done.